lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -0,0 +1,92 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'js/views/message'],
+        function (gettext, $, _, Backbone, MessageView) {
+
+        return Backbone.View.extend({
+
+            errorIcon: '<i class="fa fa-fw fa-exclamation-triangle message-error" aria-hidden="true"></i>',
+            errorMessage: gettext('An error has occurred. Please try again.'),
+
+            srAddBookmarkText: gettext('Click to add'),
+            srRemoveBookmarkText: gettext('Click to remove'),
+
+            events: {
+                'click .bookmark-button': 'bookmark'
+            },
+
+            initialize: function (options) {
+                this.bookmarked = options.bookmarked;
+
+                this.errorMessageView = new MessageView({
+                    el: $('.coursewide-message-banner'),
+                    templateId: '#message_banner-tpl'
+                });
```

Comment:
Since this will happen so rarely, can we create a method `showError` here and only initialize this when needed?